src/Tasks/Culture.cs

```diff
@@ -38,9 +39,11 @@ string dependentUponFilename
 
             if (String.Equals(Path.GetFileNameWithoutExtension(parentName),
                                    Path.GetFileNameWithoutExtension(name),
-                                   StringComparison.OrdinalIgnoreCase))
+                                   StringComparison.OrdinalIgnoreCase) ||
+                                   retainCultureInResourceName)
             {
-                // Dependent, but we treat it is as not localized because they have same base filename
+                // Dependent or explicitly using culture as part of the file name
```

Comment:
This comment and "retainCultureInResourceName" don't really make sense to me.

Reply:
I changed the name to `treatAsCultureNeutral` since all behavior of that particular file would be such that it had no culture. Calling it something close to 'keep the culture in the name' is the end result, but the new name should provide a bit more context as to what's happening to the file. Also added comments to explain that as a result of setting that boolean to true, it will keep the culture within the file name.